.travis.yml

```diff
@@ -1,16 +1,27 @@
-os:
-  - linux
-  - osx
 language: generic
 sudo: required
 dist: trusty
-osx_image: xcode9.4
 
-env:
-  matrix:
-    - MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.0
-    - MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.1
-    - MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.2
+matrix:
+  include:
+    - os: osx
+      osx_image: xcode9.4 
+      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.0
+    - os: osx
+      osx_image: xcode9.4
+      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.1
+    - os: osx
+      osx_image: xcode10.1
+      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.1
+    - os: osx
+      osx_image: xcode10.1
+      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.2
+    - os: linux
```

Comment:
Would it make sense to move `dist: trusty` down to these linux entries? I assume `os: osx` simply ignores `dist:` (akin to how a linux entry might ignore `osx_image`), so maybe this doesn't matter.

Reply:
yes, it's just ignored. we'd have to repeat the same line three times, so I think we should just leave it at the top